include/pokemon.h

```diff
@@ -183,7 +185,7 @@ struct BattlePokemon
     /*0x16*/ u32 spAttackIV:5;
     /*0x17*/ u32 spDefenseIV:5;
     /*0x17*/ u32 isEgg:1;
-    /*0x17*/ u32 abilityNum:1;
+    /*0x17*/ u32 abilityNum:2;
```

Comment:
Maybe ghoul knows something I don't but this also seems like it doesn't align as intended. Maybe slap these at the end or cannibalize the `unknown` field that seems unused?

Reply:
same thing as above. Won't touch these bits. Even though this one here at least makes sense (5+1+2=8)